### PR TITLE
Add Test case for eslint config 

### PR DESCRIPTION
**Pull request by n3tr**
Currently, It's a bit hard to test a changes we made to config files. This might be help

```sh
yarn
yarn test
```

Passed
screen shot 2017-08-01 at 5 41 43 pm

Failed
screen shot 2017-08-01 at 5 42 35 pm

### CODE REVIEW

File: test/validate-config.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+//   var cli = new CLIEngine(config)
+
+    
+
+//   var report = cli.executeOnText('var obj = { key1: "value",\n key2: "value",}')
+//   console.log(report.results[0].messages)
+
+//   t.end()
+// })
```
Comment (groupdoll) [inline test/validate-config.js]: Should we delete this unused code?

Comment (n3tr) [inline test/validate-config.js]: Indeed! I do.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    working_directory: ~/eslint-jitta
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
```
Comment (nuttt) [inline .circleci/config.yml]: maybe we better use checksum on yarn.lock?

**Comment by n3tr** [comment]
@nuttt @groupdoll done 😅

**Pull request merged**